db/seeds.rb

```diff
@@ -1,3 +1 @@
-User.destroy_all
-
 User.create(username: 'Dice Master')
```

Comment:
We might need to remove this as well or else we'll get a validation error thrown at us every time we deploy to the production since there's already the one an only Dice Master in our prod DB.

Reply:
I'll change this to `find_or_create_by!()` and it should fix any issues we would run into. I'm also gonna add a character etc. so we have data for all our current models.